src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
Pay attention to access modifiers. Classes, fields, constructors, methods must have access modifiers otherwise default will be used that isn't a good practice. In your case, the `ColorSupplier` class should be declared as `public`.